### ISSUE TITLE
[SPIKE] [Campaign Landing Pages] [UX] Audit stakeholder requests to determine quick-wins

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#19446]
## Status
[2025-01-15] [Fran] [Slack message](https://dsva.slack.com/archives/C06EK4NLGDQ/p1736953200454919) to Michelle asking for the info Dave shared with her so tickets can be created. Turning this into a Spike for historical purposes and will close once the info has been added.

## Description

Stakeholders (Justin/Danielle T/VHA/OPIA) have shared areas of CLP creation that have posed roadblocks to their usage. 

See list below, as well as the [Initiative](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/campaign-landing-page/2024-Campaign%20Landing%20Page%20V2%20Enhancements%20Initiative.md).

This ticket is for UX (Dave P) to identify top non-engineering opportunities and quick wins. 

### Stakeholder feedback
(n) indicates the number of stakeholders who mentioned this feedback 

**Hero banner panel**
- intro needs expanded character count (2)
  - 3rd stakeholder indicated she usually does fine with existing limit but sometimes needs just another 4 or 5 characters
- use of at least 1 CLP was not been approved by stakeholders because the segment is limited to a single CTA
- stakeholders question the need for any CTA in this segment (although [original template requirements](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/campaign-landing-page#the-template-requirements-are) are rigid with locations for primary and secondary CTAs) (2)

**Why this matters**
- needs expanded character count to fully provide context (2)
  - 3rd stakeholder indicated she usually does fine with existing limit but sometimes needs just another 4 or 5 characters
- title can be perceived as insulting or presumptive - stakeholders recommend a change in title, a consideration for making it editable, or allowing editors to choose among group of titles (1)
- feels duplicative to have both `Why this matters` and `What you can do` - editors and stakeholders would like to be able to choose to use one of the two and allow these two segments to be reordered (2)
- Selection of audiences should not be required (2), should allow selection of more than 3 (3), and list of choices needs to be expanded (3)
  - Survivors as suggested audience

**What you can do**
- "promo" name is misleading and confusing to editors (2)
- editors need more than 3 promo boxes or the ability to add more than one of these segments (3)
  - 1 editor indicated she manipulates the sections to meet the need - sometimes a video can be put in a spotlight, it doesn't have to always be in the video section
- editors need the ability to reorder segments and/or to add more than one of a given segment type (2)

**Connect with us panel**
- only includes four administrations (VBA, VHA, NCA or "VACO")

**Editorial workflow**
- editors need to be able to preview as they create content (possibly for an individual segment) (3)
  - 1 editor mentioned it would be nice to preview without the need to first save as draft for efficiency and to keep down the number of revisions
- editors need a way to navigate the page and move from segment to segment (1)
- nesting of forms within forms is confusing (1)
- link teasers should not always be required (2)

**Other feedback**
- need to be able to point to PDF via CTA
- criteria for creation isn't clear and/or needs to be expanded for great use cases
- should be "VA.gov" rather than "VACO"
- access should be available at VISN level or lower
- would be good to get report of outcomes/metrics
- forcing a CTA so earlier may be a disservice - sometimes more context/content is needed before a call to action is appropriate
- may need to be less restrictive on PDFs - or make it easier to create a collection of resources (outreach hub?) and then point to it 

**Tips and tricks shared by editors**
- 1 editor searches VA news and then shares the link as a way to provide a pre-filtered search for "More stories"

### User story

**AS A** PO/PM/UX lead managing CLPs
**I WANT** to know of any quick-wins possible that don't utilize engineering resources
**SO THAT** the CLPs will be more widely used, more frequently, and with greater success.

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

This analysis will have no impact; purely investigation at this point.

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
* **Functional testing:** N/A
* **Code review:** N/A
* **Design review:** N/A
* **Accessibility review:** N/A
* **Product acceptance:** N/A

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [ ] From the list above, identify top non-engineering opportunities/quick wins
- [ ] Share outcomes/feedback with @aklausmeier 
- [ ] Share outcomes with Fran Cross and Michelle Middaugh
- [ ] Close ticket

**Comment by jilladams** [comment]
@aklausmeier right now this doesn't fit into sprint 15 given other UX priorities (DUW usability research, R&S search research plan, and CMS design required to finish out phone number migration to ship).

**Comment by jilladams** [comment]
From UX sync: this ticket is lower priority than R&S research. Removing from Sprint 16.

**Comment by jilladams** [comment]
Per planning conversation: 
* The notes in this ticket body capture the review of findings from a Michelle / Dave convo in October. 
* The audit itself is basically complete, via those notes.
* Next step: @FranECross to review findings with Michelle, ticket the work to implement, and put those new tix through refinement.

**Comment by mmiddaugh** [comment]
@FranECross 

### Quick/non-engineering wins 

**Hero banner panel**
   - Expand the intro character count limitation from 100 to 160 characters 
     - Rationale: Adding 50 characters gives CLP editors the few extra characters they expressed would be helpful while being low enough to be used as the meta description in search results without being truncated. Pending federal standards for meta page description recommend character counts between 50 and 160. 
   - Change the primary CTA from required to optional. However, if the editor chooses to add a primary CTA, both the link and link text are required
   - Rationale: Editors indicated that sometimes more context/content is needed before a call to action is appropriate 
    
**Why this matters**
  - Expand the intro character count from 350 to 450
     - Rationale: CLP editors are prompted to use this section for the most impactful information yet editors reported the existing character limitation prevented them from adding context which is sometimes needed. 
 - Expand the audience cardinality from 3 to 5 audiences 
    - Rationale: allowing the selection of 5 audiences gives more flexibility without severely impacting the front end design 

**Connect with us** 

If Dave can, the "Connect with us" block should have "Veterans Affairs" as an option, rather than VACO. 
 - we should test to see what renders on the front end

**Change management**
- 
- Separately, we'll want to update the KB articles with more examples, scenarios, and tips. We can use best practices to reinforce how the template should be used, even if the template doesn't enforce compliance
- announcements to editors about changes

### For the future

- Revisit `Why this matters` and `What you can do` sections 
  -  I don't think we can remove the requirement on these sections without FE work to ensure the headings do not render unless the content fields are populated.  
- Align the audience list with the audience taxonomy for consistency (one finalized following R&S research)
- We'll need Design to evaluate the impact of changes to the number of promo boxes and/or the absence of link teasers
- Engineering will be needed to enable previewing, make navigational flow improvements or adjust nesting forms

**Comment by FranECross** [comment]
Closing; tickets will be created based on Michelle's note in the comment above.
